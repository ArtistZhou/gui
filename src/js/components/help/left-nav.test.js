import React from 'react';
import renderer from 'react-test-renderer';
import LeftNav from './left-nav';

it('renders correctly', () => {
  const tree = renderer.create(<LeftNav />).toJSON();
  expect(tree).toMatchSnapshot();
});
