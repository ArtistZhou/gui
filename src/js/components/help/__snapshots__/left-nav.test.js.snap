// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftNav Component renders correctly 1`] = `
<ul
  className="MuiList-root leftFixed MuiList-padding"
>
  <li
    className="MuiListSubheader-root MuiListSubheader-gutters"
  >
    Help topics
  </li>
</ul>
`;
